### ISSUE TITLE
Update bootstrap to select between GRCh37/38

### ISSUE THREAD

**Issue by robinandeer** [robinandeer/chanjo#166]
Latest CCDS is based on GRCh38

**Comment by robinandeer** [comment]
Links:

- https://s3.eu-central-1.amazonaws.com/chanjo/ccds.15.grch37p13.bed.zip
- https://s3.eu-central-1.amazonaws.com/chanjo/ccds.18.grch38p2.bed.zip

**Comment by biocyberman** [comment]
Yes, I fumbled for a while with `chanjo init` and `chanjo db setup` thinking that I can get the directory ready to work with, but it turns out that `chanjo bootstrap` does all the work. I don't know when to use the two previous command then. And also I found that chanjo is currently support only GRCH37. It is may be worth to point that out in the documentation or FAQ. Last but not least, it seems that `chanjo` developer intends to generate the bed file from CCDS by them self and publish it with chanjo. This is both a good thing and a limiting factor. For example, I don't see a plan to support other organisms of user choices.

**Comment by biocyberman** [comment]
Okay, maybe I got it after peeking into the code further :) 
If I work with GRCh37, I run `chanjo bootstrap` directly and ignore `init`, `db setup` and `link` commands.

If I work with other genomes, or bed files, I have to through the set of initialization commands, namely `init`, `db setup`, `link` before I can do `calculate` and other query commands.

I will keep the previous comment to show what a normal user may encounter.
